hack/ocm-minikube-ramen.sh

```diff
@@ -85,18 +85,38 @@ ramen_deploy()
 	a
 }
 exit_stack_push unset -f ramen_deploy
+ramen_deploy_hub()
+{
+	ramen_deploy $1 $2 hub
+}
+exit_stack_push unset -f ramen_deploy_hub
+ramen_deploy_spoke()
```

Comment:
Based on this we can remove [this](https://github.com/RamenDR/ramen/blob/855fea5c4c8f9f4d2fcdfb91d49a3cc5de39be78/hack/ocm-minikube.sh#L165) line in ocm deploy, as we do not need ManifestWork CRD on the spoke clusters anymore. (as drpc is on the hub only and has the dependency with the said CRD)

Reply:
Thanks!  It is removed in pull request https://github.com/RamenDR/ramen/pull/171